drivers/clock_control/clock_control_mcux_ccm_rev2.c

```diff
@@ -106,8 +106,28 @@ static int mcux_ccm_get_subsys_rate(const struct device *dev,
 	default:
 		return -EINVAL;
 	}
-
+#ifdef CONFIG_SOC_MIMX93_A55
```

Comment:
As we want to use ccm rev2 driver for i.mx93, let's also revert the update in ccm rev1 driver.

Reply:
done.